src/runtime/virtcontainers/pkg/mock/mock.go

```diff
@@ -29,6 +31,17 @@ func GenerateKataMockHybridVSock() (string, error) {
 	return fmt.Sprintf(testKataMockHybridVSockURLTempl, dir), nil
 }
 
+func RemoveKataMockHybridVSock(sockAddress string) error {
+	var sockPrefix = "mock://"
```

Comment:
Nit: It would be safer to make this a top-level `const` and use it to construct `testKataMockHybridVSockURLTempl`